delegatedSending.js

```diff
@@ -77,7 +87,7 @@ module.exports.getTotalInvites = async function(contracts, point) {
  * Generate a list of planets for as to send as invites
  * NOTE that the returned list isn't guaranteed to contain exactly amount items,
```

Comment:
I forgot to mention this before as well, but technically this NOTE in the docstring should probably be better-formatted, since these things are designed to render nicely via JSDoc.

Which reminds me.. I should immediately tweak `package.json` to rebuild the docs before every publish.  The API docs link in the README is an [embarrassment](https://github.com/urbit/azimuth-js/blob/master/docs/index.md).

Reply:
Ah, of course. Might just need to be `Note:` or w/e instead then? Thanks!